test/views/p5/NewSealNumberViewSpec.scala

```diff
@@ -42,7 +42,7 @@ class NewSealNumberViewSpec extends InputTextViewBehaviours[String] {
 
   behave like pageWithHeading()
 
-  behave like pageWithCaption(mrn.toString)
```

Comment:
Could we maybe make the scond param optional? and make the default 'MRN - ' as I dont think this is going to change much.